test/parachain/src/command.rs

```diff
@@ -210,10 +209,11 @@ pub fn run() -> Result<()> {
 					.chain(cli.relaychain_args.iter()),
 			);
 
-			polkadot_cli.base_path = cli.run.base_path()?.map(|x| x.join("polkadot"));
-
 			runner.run_full_node(
 				|config| {
+					polkadot_cli.base_path =
+						config.base_path.as_ref().map(|x| x.path().join("polkadot"));
+
```

Comment:
The base_path had to move here because the temporary directory is created when the configuration is created.